src/routes/(modeles-services)/services/[slug]/orienter/merci/+page.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import LinkButton from "$lib/components/display/link-button.svelte";
+  import { arrowLeftLineIcon, flashLightIcon } from "$lib/icons";
+  import Layout from "../layout.svelte";
+  import type { PageData } from "./$types";
+  import { orientation } from "../store";
+  import illustration from "$lib/assets/illustrations/illu-favs.svg";
+  import Notice from "$lib/components/display/notice.svelte";
+
+  export let data: PageData;
+</script>
+
+<Layout {data}>
+  <div
+    class="gap-s46 px-s14 mt-s56 flex flex-col-reverse items-center justify-around rounded-md border border-gray-03 py-s32 lg:flex-row"
+  >
+    <div class="max-w-lg basis-1/2 text-center">
+      <div class="mx-auto mb-s12 h-s24 w-s24">
+        {@html flashLightIcon}
+      </div>
+      <h2 class=" font-bold text-gray-text">
+        Votre demande a été transmise à la structure porteuse du service
+      </h2>
+      <p class="legend">
+        Vous allez recevoir une copie de cette demande par e-mail, dans quelques
+        instants.
+        {#if $orientation.beneficiaryContactPreferences?.includes("email")}
```

Comment:
```suggestion
        {#if $orientation.beneficiaryContactPreferences.includes("email")}
```
Normalement ont est sûr que c'est un array, si le store est initialisé correctement (ce qui n'est pas encore le cas dans le code commenté de store.ts, donc à corriger!).

Par contre je pense qu'il faut qu'on précise avec Andrei: est-ce qu'on n'envoie pas par email même si c'est pas une des préférences, à partir du moment ou on a un email?

Reply:
Tu as soulevé le point avec Andrei du coup?